src/main/java/bitflow4j/steps/misc/SleepOnTagChange.java

```diff
@@ -0,0 +1,43 @@
+package bitflow4j.steps.misc;
+
+import bitflow4j.AbstractPipelineStep;
+import bitflow4j.Sample;
+import bitflow4j.script.registry.Description;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @author kevinstyp
+ */
+@Description("Whenever the given onChangedTag-Tag changes, this step sleeps for the given time in ms.")
+public class SleepOnTagChange extends AbstractPipelineStep {
+
+    protected static final Logger logger = Logger.getLogger(SleepOnTagChange.class.getName());
+
+    private final int time;
+    private final String onChangedTag;
+
+    private String previousTag;
+
+    public SleepOnTagChange(int time, String onChangedTag) {
```

Comment:
I suggest more consice parameter names: milliseconds and tag
"on changed" is already in the step name, and the time duration is not clear.